src/After.tsx

```diff
@@ -88,15 +93,14 @@ class Afterparty extends React.Component<AfterpartyProps, AfterpartyState> {
     const initialData = this.prefetcherCache[location.pathname] || data;
```

Comment:
I think data also should be pointing to `previousLocation`
```suggestion
     const initialData = this.prefetcherCache[(previousLocation || location).pathname] || data;
```

Reply:
yeap, thanks.